benchmarks/simulator.py

```diff
@@ -45,3 +46,26 @@ def time_scipy_lsoda_no_compiler_directives(self):
 
     def time_cupsoda(self):
         self.sim_cupsoda.run()
+
+
+class SSASchlogl(object):
```

Comment:
Should be `SSASchloegl` (with an 'e') to match the others

Reply:
Yep! Fixed.